core/controller/src/main/scala/whisk/core/controller/Actions.scala

```diff
@@ -218,7 +219,24 @@ trait WhiskActionsApi extends WhiskCollectionAPI {
         parameter('overwrite ? false) { overwrite =>
             entity(as[WhiskActionPut]) { content =>
                 val docid = DocId(WhiskEntity.qualifiedName(namespace, name))
-                putEntity(WhiskAction, entityStore, docid, overwrite, update(user, content)_, () => { make(user, namespace, content, name) })
+                def doput = {
+                    putEntity(WhiskAction, entityStore, docid, overwrite,
+                        update(user, content)_, () => { make(user, namespace, content, name) })
+                }
+
+                content.exec match {
+                    case Some(seq @ SequenceExec(_, components)) =>
+                        info(this, "checking if sequence components are accessible")
```

Comment:
This may succeed at creation time and the access rights could still be different at runtime, right? So it's a "don't shoot yourself in the foot" kind of check?